crawl-ref/source/dat/des/branches/lair.des

```diff
@@ -2748,6 +2778,107 @@ MAP
   -------------------
 ENDMAP
 
+NAME:   pdpol_lair_dawg_pound
+TAGS:   no_monster_gen
+DEPTH:  Lair
+KMONS:  1 = wolf / warg w:5
+KMONS:  2 = hell hound / raiju
+KITEM:  2 = *
+SUBST:  _ = x _:3, S = S:1 . %:2
+NSUBST: s = 1:S / *= . S:1, D = 2:2 / 4= 1 2:1 / *= . 1:2
+: dgn.delayed_decay(_G, 'S', "felid skeleton")
+CLEAR:  _
+MAP
+            ________
+          __xxxxxxx__
+          _xx.SS..x__
+      _____x..SS..x__
+    ___xxxxx....DDx__
+ ____xxx..SS..DDxxx__
+ _xxxx.....SxDxxxxx__
+_xxSS....x..xxxx____
+_xDSS..Dxxssxxxx__
+_xS..Dxxxssxx____
+_xxDxxxxssxxx_
+__xxxxx..xx_
+    xx@@xx
+ENDMAP
+
+# Dead by dawn!
+NAME:    pdpol_lair_cabin_in_the_woods
+TAGS:    no_item_gen no_monster_gen
+DEPTH:   Lair
+# Trying to get some good horror move trope representation
+KMONS:   M = Rupert / reaper / vampire knight / phantasmal warrior / tentacled starspawn
+# The last survivor
+KMONS:   h = human
+SHUFFLE: M/h/?
+NSUBST:  M = M / | / S / `, h = h / *%% / S / `
+NSUBST:  T = 2= 0 9:5 / *= .:3 t
+NSUBST:  a = a / c, b = b / c, d = d / c
+NSUBST:  e = e / c, f = f / c, g = g / c
+NSUBST:  abd = 2:+ / c, efg = 2:+ / c
+SUBST:   " = `:4 ., `? = `:3 ., _ = t_
+KPROP:   ` = bloody
+MARKER:  F = lua:fog_machine { cloud_type = "thin mist", pow_min = 1, \
+         pow_max = 3, delay = 10, start_clouds = 1, size_min = 2, \
+         size_max = 3, spread_rate = 33 }
+: dgn.delayed_decay(_G, 'S', "human skeleton")
+CLEAR:   _
+MAP
+    ______________________
+   ________________________
+   @ttttttttttttttttttttt__
+ @..ttttTttttttttttttttttt__
+ tTt.tt..ccccccccccccccctt__
+ttt.tTt..c?????d``ehhhhctt__
+ttTt.TtF.c?????d``ehhhhctt__
+ttt.ttt..caaaaac``ehhhhctt__
+ttTt..t."n`````b``cffffctt__
+tTt..T'T"=`````b``gMMMMctt__
+tt.ttt.F"n`````b``gMMMMctt__
+ttT.tt...c`````b``gMMMMctt__
+ tTt.tt..ccccccccccccccctt__
+ @..ttttTttttttttttttttttt__
+   @ttttttttttttttttttttt__
+   ________________________
+    ______________________
+ENDMAP
+
+NAME:   pdpol_lair_cavern_descent
+TAGS:   no_pool_fixup
+DEPTH:  Lair
+KMONS:  1 = hydra / boulder beetle / hornet / rime drake
+KMONS:  2 = cane toad / black mamba / komodo dragon / polar bear
+KMONS:  3 = water moccasin / scorpion / black bear / wolf / bullfrog
+KMONS:  4 = basilisk / torpor snail w:5 / manticore w:3
+KMONS:  5 = electric eel
+SUBST:  u = wW
+NSUBST: ' = 2:1 / 6= 1:2 2:3 3 / .
+NSUBST: W = 3:4
+NSUBST: w = 2:5
```

Comment:
Recommend some amount of randomization on the number of monsters being placed here.

Reply:
Any thoughts on the loot distribution for this one (and the rest in general)? That's an area where I feel mostly like I don't know enough to get too creative so I kept it pretty generic.